test/unit/math/prim/arr/meta/is_vector_test.cpp

```diff
@@ -8,6 +8,6 @@ TEST(MetaTraits, is_vector) {
 
   EXPECT_TRUE(is_vector<std::vector<double> >::value);
   EXPECT_TRUE(is_vector<std::vector<int> >::value);
-  EXPECT_TRUE(is_vector<std::vector<const double> >::value);
-  EXPECT_TRUE(is_vector<std::vector<const int> >::value);
+  EXPECT_TRUE(is_vector<const std::vector<double> >::value);
+  EXPECT_TRUE(is_vector<const std::vector<int> >::value);
```

Comment:
@syclik fyi I changed these because it looks like having `const` types in C++11 is illegal?

https://stackoverflow.com/questions/6954906/does-c11-allow-vectorconst-t